static/app/components/events/eventReplay/replayInlineOnboardingPanelBackend.tsx

```diff
@@ -13,6 +13,7 @@ import {IconBroadcast} from 'sentry/icons/iconBroadcast';
 import {t, tct} from 'sentry/locale';
 import type {PlatformKey} from 'sentry/types';
 import {useReplayOnboardingSidebarPanel} from 'sentry/utils/replays/hooks/useReplayOnboarding';
+import ToggleButton from 'sentry/views/issueDetails/toggleButton';
```

Comment:
Let's call this something like `SectionToggleButton` to be more specific about what it's for. "Toggle" to me only means "a switch between 2 states", and doesn't specifically connote "expand"/"collapse".

Reply:
ah that's my bad, i had the component named `IssueDetailsToggleButton` to be specific but forgot to import it as such - but can rename it to `SectionToggleButton`